.github/workflows/ci.yml

```
@@ -52,6 +52,15 @@ jobs:
     needs: test
     steps:
     - uses: actions/checkout@v4
+    - name: Set up Go
+      uses: actions/setup-go@v4
```

actionlint (1.7.10)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)